jsapp/xlform/src/view.choices.templates.coffee

```diff
@@ -1,7 +1,7 @@
 module.exports = do ->
 
   addOptionButton = () ->
-      """<div class="card__addoptions">
+      """<div class="card__addoptions js-card-add-options">
```

Comment:
`js-` classes are a widely accepted way of naming a selector that is only for JS code and should not have any stylings on it. This is made so, because if you have a class name with some styles, e.g. `card__label` and it both has some styles and is being used in JS, and some UI update comes and styles are dropped, you could drop `card__label` without noticing that you broke JS.

Reply:
that makes a lot of sense, it sounds pretty much mandatory with working with jquery